app/bundles/CoreBundle/Config/services.php

```diff
@@ -46,6 +46,7 @@
 
     $services->alias(GuzzleHttp\Client::class, 'mautic.http.client');
     $services->alias(Psr\Http\Client\ClientInterface::class, 'mautic.http.client');
+    $services->alias(Symfony\Component\DependencyInjection\ContainerInterface::class, 'service_container');
```

Comment:
Just for my understanding: why this was needed?

Reply:
This service alias is used in several places but it's not available from Symfony itself anymore.